include/svgpp/parser/grammar/color.hpp

```diff
@@ -135,11 +171,21 @@ class color_grammar: public qi::grammar<Iterator, typename ColorFactory::color_t
     return ColorFactory::create(r, g, b);
   }
 
+  static color_type absolute_rgba_components(unsigned char r, unsigned char g, unsigned char b, unsigned char a)
```

Comment:
What is the spec for the alpha value? It isn't SVG 2.0/CSS 3 as in https://www.w3.org/TR/css-color-3/#alphavaluedt

Reply:
In CSS Color 3 `<alphavalue>` has 0.0 - 1.0 range. And in your PR it's integer. That is why I'm asking.

Reply:
Good catch, working on updating our copy first, then we'll make the corresponding changes here!

Reply:
Updated our copy, correcting to float here as well.